package.json

```diff
@@ -345,7 +345,7 @@
 		"photon": "^3.0.0",
 		"postcss-cli": "^6.1.3",
 		"postcss-custom-properties": "^9.1.1",
-		"prettier": "npm:wp-prettier@2.0.5",
+		"prettier": "npm:wp-prettier@2.1.1-beta-1",
```

Comment:
Could you update it in `test/e2e/package.json`? (or remove it, not sure why it is declared there)

Reply:
Yes, updated. The `test/e2e` folder has its own script with `pre-commit-hook` that does Prettier formatting. Probably a relict of the era when it was a separate repo and later had its own `node_modules` tree and lockfile. Worth a cleanup in another PR.